.github/workflows/publish.yml

```
@@ -20,16 +20,20 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      # v1.1.0
-      - uses: haya14busa/action-cond@c69871f4d4693e93126e770d7c3588cb1d473462
-        id: source-value
-        with:
-          cond: ${{ inputs.source == '' }}
-          if_true: "https://api.nuget.org/v3/index.json"
-          if_false: "${{ inputs.source }}"
-
       - name: Print source
-        run: echo NuGet Source:${{ steps.source-value.outputs.value }}
+        shell: pwsh
+        run: |
+          $Source = @'
+          ${{ inputs.source }}
```

I thought this would work:

```suggestion
${{ inputs.source || 'https://api.nuget.org/v3/index.json' }}
```

Have you tried that?

---

Actually if the input is not provided, [it already defaults to](https://github.com/Lombiq/GitHub-Actions/pull/14/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R9) `https://api.nuget.org/v3/index.json`, so there is no real point to this and I'm just going to remove it. If the consumer wisely gave this parameter null or empty string value, then we shouldn't mask the error.

---

Agreed. KISS. No offense.

---

😘